src/app/Models/Traits/SpatieTranslatable/HasTranslations.php

```diff
@@ -80,7 +80,13 @@ public static function create(array $attributes = [])
         // do the actual saving
         foreach ($attributes as $attribute => $value) {
             if ($model->isTranslatableAttribute($attribute)) { // the attribute is translatable
-                $model->setTranslation($attribute, $locale, $value);
+                if (is_array($value)) {
+                    foreach ($value as $lang => $val) {
+                        $model->setTranslation($attribute, $lang, $val);
+                    }
```

Comment:
```suggestion
                    $model->setTranslations($attribute, $value);
```

This works as well.

Reply:
@kolaente can you please commit this change? I think you can do it directly here on GitHub ✌

Reply:
Done!